code/src/App.js

```diff
@@ -1,9 +1,31 @@
-import React from 'react';
+import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+// import NewTask from 'components/NewTask'
+import { OuterWrapper, InnerWrapper } from 'components/styling/Wrappers'
+import TodoList from 'components/TodoList'
+import todo from 'reducers/todo'
+import NewTodo from 'components/NewTodo'
+import { Header } from 'components/Header'
+
+const reducer = combineReducers({
+  todo: todo.reducer
+})
+
+const store = configureStore({ reducer })
+
+store.subscribe(() => localStorage.setItem('todos', JSON.stringify(store.getState().todo.items)))
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  );
-}
+    <Provider store={store}>
+      <OuterWrapper>
+        <InnerWrapper>
+          <Header />
+          <TodoList />
+          <NewTodo />
```

Comment:
Suggestion: From a ux perspective I would switch TodoList and NewTodo so the input field doesn't move each time you add a task. Perhaps also move check box so u have check and delete buttons on each side of the text. You want to keep them far away so you don't press delete by mistake. (I have done the same with delete all/check all... 😬)